### DISCUSSION TITLE
一些使用说明

### DISCUSSION THREAD

**Discussion by sakuraskyteam** [imsyy/home#61] [Announcements]
能不能出一个食用说明，小白表示不会改，拜托！

**Comment by imsyy** [comment]
如果需要调整配置，可以直接修改根目录下的 `env` 文件（文件中都有相对应的注释），如果想修改源代码，可能需要学习一些关于 `Vue` 的知识

如果不知道如何部署
- 安装 `yarn`（以管理员模式运行 cmd，输入 `npm install --global yarn`）
- 安装依赖（在项目根目录中启动 cmd，输入 `yarn install`）
- 构建静态文件（继续在项目根目录的 cmd 中输入 `yarn build`）
- `dist` 目录下的文件就可部署到任意支持静态网站的平台了

**Reply by Aaron-zsy** [reply]
请问如何更新？

**Reply by imsyy** [reply]
git pull

**Reply by losie** [reply]
大佬等于说 是先修改目录env后 部署dist 上传目录？

**Reply by imsyy** [reply]
对的，或者可以使用 vercel，直接一键部署

**Reply by losie** [reply]
谢谢！

**Reply by Onionbai** [reply]
logo字母yougetit的u显示的和其他不一样，有什么办法把它调成一样吗？感觉好像只有几个特定的字母会以花体展示出来

**Reply by Onionbai** [reply]
![image](https://user-images.githubusercontent.com/76639100/218314231-20368298-0357-46e9-aed8-f900ed49c4a0.png)
最后一个U，我服务器上传4.0版本也是加载不出来，就用的3.13

**Reply by imsyy** [reply]
> ![image](https://user-images.githubusercontent.com/76639100/218314231-20368298-0357-46e9-aed8-f900ed49c4a0.png)
> 最后一个U，我服务器上传4.0版本也是加载不出来，就用的3.13

下载介绍中的完整字体并替换

**Comment by Malong200424** [comment]
宝塔里面是不是不能搭建？

**Reply by imsyy** [reply]
如果服务器运行内存过低就不行（我也不知道为啥 😂）

**Reply by Onionbai** [reply]
好奇怪哎，之前搭建4.0新版在768M内存上是没法运行的，这几天心血来潮，熟悉了正确的nodejs的安装方法，在本地vmvps yarn install完成之后，再上传dist到网页目录，竟然可以用了。。。

**Reply by ghost** [reply]
可以发我一份吗 一直报错
error Couldn't find a package.json file in "/home"
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Reply by Onionbai** [reply]
https://www.123pan.com/s/tcLA-uIDKv.html

**Reply by Onionbai** [reply]
> 可以发我一份吗 一直报错 error Couldn't find a package.json file in "/home" info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
能用了吗

**Reply by ghost** [reply]
我仔细研究了一下文档重新编译成功了 也麻烦hxd了

**Comment by yanxscn** [comment]
出个宝塔的完整教程吧  研究好几天了

**Reply by imsyy** [reply]
宝塔 - 网站 - Node项目
![1677811960136](https://user-images.githubusercontent.com/42232682/222619490-4d7a6f6f-9767-4b98-9c01-7b382ddae910.png)

**Reply by yanxscn** [reply]
弄完之后加载老是转圈  加载不进去啥情况

**Reply by 176822** [reply]
求作者更新宝塔教程
这样子还是不明白

**Reply by imsyy** [reply]
> 求作者更新宝塔教程 这样子还是不明白

https://github.com/imsyy/home/discussions/104

**Reply by imsyy** [reply]
> 出个宝塔的完整教程吧 研究好几天了

https://github.com/imsyy/home/discussions/104

**Reply by chenyu614** [reply]
yarn install后报错如下，怎么解决呀大佬
warning vue > @vue/compiler-sfc > magic-string > sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
warning unplugin-auto-import > magic-string > sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
warning vite-plugin-pwa > workbox-build > rollup-plugin-terser@7.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-terser

**Reply by imsyy** [reply]
> yarn install后报错如下，怎么解决呀大佬 warning vue > @vue/compiler-sfc > magic-string > sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead warning unplugin-auto-import > magic-string > sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead warning vite-plugin-pwa > workbox-build > rollup-plugin-terser@7.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-terser

这些不是重要的报错，都是一些依赖包过期的报错

**Reply by chenyu614** [reply]
好的，谢谢大佬回复。已经解决了，换了Ubuntu就好了。（本来是Windows 10）

**Comment by zquend** [comment]
我该怎么替换中文字体呢，我把下载好的中文字体放到/host/font/里面了，但是ctrl F5之后还是不起作用，请求无名大佬解答

**Reply by Onionbai** [reply]
#103

**Comment by xliuwl** [comment]
![bug](https://github.com/imsyy/home/assets/170300752/e68979ab-b9f6-4feb-bc3a-25af4846d0bc)
这个logo我改成中文字体太大显示不了，字体怎么改小

**Reply by xliuwl** [reply]
最后一个字显示不了

**Comment by xliuwl** [comment]
![bug1](https://github.com/imsyy/home/assets/170300752/6adf121f-96a3-4443-a8e5-c8cfc22b2cdc)
这个di地方怎么弄，把这个一点去了，怎么才能做到

**Comment by 1024078948** [comment]
音乐的api地址有些地区要挂梯才能打开   不挂梯404   https://api-meting.imsyy.top/api?server=netease&type=playlist&id=391564201  大佬可以更新一下你的api吗
